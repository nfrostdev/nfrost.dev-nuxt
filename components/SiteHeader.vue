<template>
  <header class="header">
    <logo class="logo"/>
  </header>
</template>

<script lang="ts">
  import Vue from 'vue'
  import Logo from '../components/Logo'

  export default Vue.extend({
    components: {
      Logo
    }
  })
</script>

<style lang="scss">
  .header {
    @apply flex justify-center items-center w-full py-6;

    @media(min-width: 48em) {
      @apply pb-0 pt-16;
    }
  }

  .logo {
    @apply h-8;

    @media(min-width: 48em) {
      @apply h-12;
    }
  }
</style>
