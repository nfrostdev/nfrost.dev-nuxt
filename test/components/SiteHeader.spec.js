import {shallowMount} from '@vue/test-utils'
import SiteHeader from '../../components/SiteHeader'

const factory = () => {
  return shallowMount(SiteHeader, {});
};

describe('SiteHeader', () => {
  test('renders properly', () => {
    const wrapper = factory();
    expect(wrapper.html()).toMatchSnapshot();
  })
})
