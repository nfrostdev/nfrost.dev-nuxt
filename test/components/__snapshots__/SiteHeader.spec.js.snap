// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SiteHeader renders properly 1`] = `
"<header class=\\"header\\">
  <logo-stub class=\\"logo\\"></logo-stub>
</header>"
`;
